nrpy/infrastructures/CarpetX/general_relativity/T4DD_to_T4UU.py

```diff
@@ -0,0 +1,157 @@
+"""
+Generates function to raise the stress-energy tensor using the BSSN variables.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+import sympy
+
+import nrpy.c_codegen as ccg
+import nrpy.c_function as cfc
+import nrpy.grid as gri
+import nrpy.indexedexp as ixp
+import nrpy.helpers.parallel_codegen as pcg
+
+import nrpy.infrastructures.CarpetX.simple_loop as lp
+import nrpy.equations.general_relativity.g4munu_conversions as g4conv
+
+standard_ET_includes = [
```

Comment:
Centralize

Reply:
Centralized in `CarpetX_include_header.py`